src/spyglass/common/common_ephys.py

```diff
@@ -132,15 +136,14 @@ def make(self, key):
                     "region_id": BrainRegion.fetch_add(
                         region_name=elect_data.group.location
                     ),
-                    "x": elect_data.x,
-                    "y": elect_data.y,
-                    "z": elect_data.z,
-                    "filtering": elect_data.filtering,
-                    "impedance": elect_data.get("imp"),
                     **electrode_constants,
                 }
             )
 
+            for f in optional_fields:
+                if f in elect_data:
+                    key.update({f: elect_data.get(f)})
+
```

Comment:
If using `get` or `getattr`, no need for this 
```suggestion
```

Reply:
But then if `f` doesn't exist in `elect_data`, wouldn't `key[f]` end up being set to `None`? I think I just don't want `key` to have any key called `f` (instead of adding a key-value pair `f`-`None`)

Reply:
Having `None` values for x, y, z will have the same result as not supplying them when you insert - they'll be NULL. For filtering, you can do `elec_data.get('filtering', 'unfiltered')` and remove the need for altering the table

Reply:
Cool, yes made that change